metrics/core/src/main/java/org/hyperledger/besu/metrics/MetricsConfigurationModule.java

```diff
@@ -11,27 +11,23 @@
  * specific language governing permissions and limitations under the License.
  *
  * SPDX-License-Identifier: Apache-2.0
- *
  */
 
-package org.hyperledger.besu.evmtool;
+package org.hyperledger.besu.metrics;
 
-import org.hyperledger.besu.metrics.MetricsSystemFactory;
 import org.hyperledger.besu.metrics.prometheus.MetricsConfiguration;
-import org.hyperledger.besu.plugin.services.MetricsSystem;
 
 import javax.inject.Singleton;
 
 import dagger.Module;
 import dagger.Provides;
 
-@SuppressWarnings("WeakerAccess")
```

Comment:
Intrigued to know why you were able to delete this

Reply:
It moved packages, now the module is in the same package that MetricsConfiguration is in.